src/main/kotlin/pl/edu/agh/gem/external/dto/report/ReportNotificationRequest.kt

```diff
@@ -0,0 +1,8 @@
+package pl.edu.agh.gem.external.dto.report
+
+data class ReportNotificationRequest(
+    val id: String,
+    val title: String,
+    val userId: String,
+    val attachmentId: String,
+)
```

Comment:
maybe groupId because downloading it from attachment store required attachmentId and groupId

Reply:
sure we can add that